druntime/src/core/attribute.d

```diff
@@ -236,10 +236,17 @@ version (UdaGNUAbiTag) struct gnuAbiTag
 {
     string[] tags;
 
-    this(string[] tags...)
+    this(scope string[] tags)
```

Comment:
```suggestion
    this(return scope string[] tags)
```
It very clearly stores the array in `this`

Reply:
implemented